@here/olp-sdk-dataservice-read/lib/client/VersionedLayerClient.ts

```diff
@@ -36,29 +36,68 @@ import {
     RequestFactory
 } from "..";
 
+/**
+ * Interface with parameters for use in the new constructor of VersionLayerClient.
```

Comment:
for use to initialize VersionLayerClient.

Reply:
fixed

Reply:
fixed